### PR TITLE
add provenance in multiexposure

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix?
* Ticket: #
* Funded by: DFAT
* Description: add provenance in multiexposure

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

**Comment by timlinux** [comment]
Nice - LGTM thanks!

**Comment by Gustry** [comment]
Thanks, you have done the review even if the PR is 'in progress' contrary to the other one yesterday ;-)

File: safe/impact_function/multi_exposure_wrapper.py [hunk 1]
```diff
@@ -294,6 +444,7 @@ def prepare(self):
                 from the code.
         :rtype: (int, m.Message)
         """
+        self._provenance_ready = False
         if len(self._exposures) < 2:  # 2 layers minimum.
             message = generate_input_error_message(
                 tr('No exposure layer provided'),
```
Comment (ismailsunni) [inline safe/impact_function/multi_exposure_wrapper.py]: Perhaps clearer error messagem like, "Needs at least 2 exposure layers"

Comment (Gustry) [reply safe/impact_function/multi_exposure_wrapper.py]: You see only the title, the description is the next line explaining they need 2 layers. I can still improve it

Comment (ismailsunni) [reply safe/impact_function/multi_exposure_wrapper.py]: Ah yes, sorry. Sometime github cut it in the middle of something. Thanks

**Pull request merged**